ReleaseNotes.md

```diff
@@ -23,6 +23,8 @@ Note: Starting 02/21/2023 we have moved to the semver versioning system for nami
   - Update telegraf to version 1.25.2
   - Fix bug wherein sidecar container was not muted by default
   - Removed labelSelector from ama-logs.yaml and helm charts
+  - Update fluent-bit to version 2.0.5
+  - Enabled multiline support in ContainerLogsV2 via configmap
```

Comment:
1. should be "for" instead for "in"  2. should be ContainerLogV2 instead of ContainerLogsV2